src/buffify.ts

```diff
@@ -51,15 +72,15 @@ function _jsonBuffify(json: Json, replace: JsonReplacer, space: string, level =
               values.push(Buffer.from(',' + lineBreak, 'utf8'))
             }
             values.push(Buffer.from(propIndentation))
-            values.push(_jsonBuffify(value, replace, space, level + 1, `${keyPath}[${i}]`))
+            values.push(_jsonBuffify(value, replace, sorter, space, level + 1, `${keyPath}[${i}]`))
           }
           values.push(Buffer.from(lineBreak + endIndentation + ']', 'utf8'))
         } else {
           values.push(Buffer.from('[]', 'utf8'))
         }
       } else {
         const obj = json as { [key: string]: Json | undefined }
-        const keys = Object.keys(json).sort()
+        const keys = Object.keys(json).sort((a, b) => sorter.bind(obj)(a, b, level, keyPath))
```

Comment:
Why does the sorter take `level` and `keyPath`?

Reply:
So you can sort levels and objects differently and to match the replacement interface, fx. below where we only want to sort params object properties in a specific order to make it more readable but the rest alphabetically.

``` json
{
   "stuff": "hello",
   "params": [{
          "name": "test",
          "afterParam": "hello"
   }]
}
```